velox/exec/tests/WindowTest.cpp

```diff
@@ -679,5 +679,62 @@ DEBUG_ONLY_TEST_F(WindowTest, reserveMemorySort) {
   }
 }
 
+TEST_F(WindowTest, NaNFrameBound) {
+  const auto kNan = std::numeric_limits<double>::quiet_NaN();
+  auto data = makeRowVector(
+      {"c0", "s0", "off0", "off1"},
+      {
+          makeFlatVector<int64_t>({1, 2, 3}),
+          makeFlatVector<double>({1.0, 1.0, 1.0}),
+          makeFlatVector<double>({1.0, 1.0, 1.0}),
```

Comment:
The offset columns are always corresponding to the same row or NaN. Would be great to make the values more interesting and include some non-Nan values as well.